editor.planx.uk/src/@planx/components/MapAndLabel/test/public.backNavigation.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { MyMap } from "@opensystemslab/map";
+import React from "react";
+import { setup } from "testUtils";
+import { it, vi } from "vitest";
+
+import { Presentational as MapAndLabel } from "../Public";
+import { point1, point2, point3 } from "./mocks/geojson";
+import {
+  previousDoubleDataProps,
+  previousSingleDataProps,
+} from "./mocks/Trees";
+import { addMultipleFeatures } from "./utils";
+
+beforeAll(() => {
+  if (!window.customElements.get("my-map")) {
+    window.customElements.define("my-map", MyMap);
+  }
+
+  const ResizeObserverMock = vi.fn(() => ({
+    observe: vi.fn(),
+    unobserve: vi.fn(),
+    disconnect: vi.fn(),
+  }));
+
+  vi.stubGlobal("ResizeObserver", ResizeObserverMock);
```

Comment:
nit: is `ResizeObserverMock` actually required by these tests / called in this file? Do these tests still pass if `beforeAll` only does the first definition step for `my-map` ?

Reply:
It fails when it isn't defined, not fully sure why. I'll need to look back at Daf setting up the test.todos

Reply:
No worries about tracking down more detailed answer for this - thanks for simply confirming it's necessary !